README.md

```diff
@@ -1587,11 +1593,13 @@ biggest chunk of tax they pay.
 🄳 **Employment taxes** are an additional kind of federal tax beyond ordinary income tax,
 and consist of Social Security and
 [Medicare taxes](https://www.irs.gov/businesses/small-businesses-self-employed/questions-and-answers-for-the-additional-medicare-tax)
-that are withheld from your paycheck.
-The Social Security wage withholding rate is **6.2%** up to the FICA wage base.
-The Medicare component is **1.45%**, and it does not phase out above the FICA wage base.
-You’ll also hear these called [**payroll taxes**](https://en.wikipedia.org/wiki/Payroll_tax) as
-they often show up on your pay stub.
+that are withheld from a person’s paycheck.
+Employment taxes are also referred to as
+[**payroll taxes**](https://en.wikipedia.org/wiki/Payroll_tax) as they often show up on employee
+pay stubs.
+
+- The Social Security wage withholding rate in 2018 is **6.2%** up to the FICA wage base.
```

Comment:
Let's include these in the definition. Also unbold the numbers per @Hopeopolis 's request.